internal/controllers/machinedeployment/mdutil/util_test.go

```diff
@@ -411,197 +420,165 @@ func TestFindNewMachineSet(t *testing.T) {
 	msCreatedExactlyInRolloutAfter.CreationTimestamp = rolloutAfter
 
 	tests := []struct {
```

Comment:
Diffed this test coverage against what we had for the dropped test for old ms. I think the following is missing:
* A case where we only have one new MS and no old MS ("Get empty old MachineSets")